tests/IntegrationTests/Config/appsettings.template.json

```diff
@@ -0,0 +1,20 @@
+{
+  "AppSettings": {
+    "Endpoints": {
+      "CreateEpisode": "CreateEpisode_URL",
+      "GetEpisode": "GetEpisode_URL",
+      "ProcessData": "ProcessData_URL"
```

Comment:
GetEpisode seems to be the only app setting that's being used, can probably leave the others off for now

Reply:
Great point, have removed the unused ones!